### PR TITLE
Add executor metrics.

### PR DESCRIPTION
**Pull request by vjkoskela**

### CODE REVIEW

**Comment by vjkoskela** [comment]
metrics-telemetry-more-jvm-metrics

File: app/actors/JvmMetricsCollector.java [hunk 1]
```diff
@@ -15,42 +15,69 @@
...
      * Public constructor.
      *
      * @param configuration Play app configuration.
+     * @param actorSystem Akka <code>ActorySystem</code>.
      * @param metricsFactory An instance of <code>MetricsFactory</code>.
      */
     @Inject
     public JvmMetricsCollector(
             final Configuration configuration,
+            final ActorSystem actorSystem,
```
Comment (BrandonArp) [inline app/actors/JvmMetricsCollector.java]: We can get the Actor's ActorSystem from context().system().  Is there a special reason you're injecting it?

File: app/actors/JvmMetricsCollector.java [hunk 2]
```diff
@@ -105,10 +134,79 @@ public String toString() {
...
+    }
+
+    private void addExecutorService(
+            final Map<String, ExecutorService> executorServices,
+            final String name,
+            final ExecutorService executorService) {
+        if (executorService instanceof scala.concurrent.forkjoin.ForkJoinPool) {
+            // NOTE: This is deprecated in Scala 2.12 which will be adopted by Play 2.6 (maybe)
+            // ^ Scala and hopefully Play will use Java's ForkJoinPool natively
+            final scala.concurrent.forkjoin.ForkJoinPool scalaForkJoinPool =
```
Comment (BrandonArp) [inline app/actors/JvmMetricsCollector.java]: Wow, that's annoying.

File: app/actors/JvmMetricsCollector.java (hunk 1 above)
Comment (vjkoskela) [inline app/actors/JvmMetricsCollector.java]: Fixed.

**Comment by vjkoskela** [comment]
@BrandonArp updated.

**Pull request merged**